doc/api/buffer.markdown

```diff
@@ -600,6 +600,46 @@ buf.includes('this', 4);
   // returns false
 ```
 
+### buf.lastIndexOf(value[, byteOffset][, encoding])
+
+* `value` String, Buffer or Number
+* `byteOffset` {Number} Default: `buf.length`
+* `encoding` {String} Default: `'utf8'`
+* Return: {Number}
+
+Identical to [`Buffer#indexOf()`][], but searches the Buffer from back to front
+instead of front to back. Returns the starting index position of `value` in
+Buffer or `-1` if the Buffer does not contain `value`. The `value` can be a
+String, Buffer or Number. Strings are by default interpreted as UTF8. If
```

Comment:
Also you should need to mention the default encoding at all since it is listed above.